config/webpack-config.js

```diff
@@ -19,17 +19,26 @@ var modernConf = {
   // so it needs to be defined here as an external script to ignore for
   // unmet dependency references.
   externals: { jquery: 'jQuery' },
-  context:   path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
-  entry:     scriptsManifest.getDirectoryMap( paths.unprocessed +
-                                              JS_ROUTES_PATH ),
+  cache: true,
+  context: path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
+  entry: scriptsManifest.getDirectoryMap( paths.unprocessed + JS_ROUTES_PATH ),
+  module: {
+    rules: [ {
+      test: /\.js$/,
+      use: [ {
+        loader: 'babel-loader?cacheDirectory=true'
+      } ],
+      exclude: /node_modules/
+    } ]
+  },
   output: {
-    path:     path.join( __dirname, 'js' ),
+    path: path.join( __dirname, 'js' ),
     filename: '[name]'
   },
   plugins: [
-    new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME ),
-    new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME,
-                                             [ COMMON_BUNDLE_NAME ] ),
```

Comment:
IIRC the redundant commons chunk calls here were for the scenario where all included scripts included a reference to a module EXCEPT common.js. In that case, since all scripts reference a module, the module's declaration should be moved to common.js automatically by webpack, instead of leaving them redundantly in each script. I tested this behavior in the old setup and the new setup and it still works as expected 👍